dask_jobqueue/core.py

```diff
@@ -406,6 +407,6 @@ def _del_pending_jobs(self):
             del self.pending_jobs[job_id]
         return jobs
 
-    def _job_id_from_submit_output(self, out):
-        raise NotImplementedError('_job_id_from_submit_output must be implemented when JobQueueCluster is '
-                                  'inherited. It should convert the stdout from submit_command to the job id')
+    @staticmethod
```

Comment:
Is this annotation really required?

I'm not sure how it will be handled if a subclass needs to override it?